### ISSUE TITLE
Launcher doesn't work with HSR 2.7

### ISSUE THREAD

**Issue by jckirton** [yaagl/yet-another-anime-game-launcher#497]
**Describe the bug**
Launcher is unable to parse HSR V2.7

**Screenshots**

**System Information (please complete the following information):**
 - macOS: 15.1.1
 - Device: MacBook Pro (2020(
 - CPU: Apple M1 Pro
 - Dedicated GPU (if any): N/A
 - RAM: 32GB
 - Wine Distribution: Crossover Wine (Don't know what specific version as it is only prompting to install the game post-update.)

**Additional context**
Updated the game, said the launcher does not yer support HSR 2.7 and to wait for an update. On subsequent launched of YAAGL, prompts to install HSR and says it failed to parse game version upon being pointed to the game files.

**Logs (if any)**

> If it's related to the launcher, upload `neutralinojs.log` in Yaagl Data Directory 

> If it's related to the game, upload `logs/game_{timestamp}.log in Yaagl Data Directory

**Comment by Sipperz** [comment]
yes like always will have to wait for the launcher update, which has NO ETA
